openlibrary/plugins/upstream/utils.py

```diff
@@ -650,7 +650,10 @@ def get_donation_include(type):
         dd = input['don']
         param = param+"&ymd="+dd
     if (type=='true'):
```

Comment:
Something seems strange here (we appear to be locally overriding the `type` builtin which we should avoid if we can)

See: https://docs.python.org/2/library/functions.html#type